rpc/client/local/rpc_test.go

```diff
@@ -0,0 +1,183 @@
+package local_test
```

Comment:
Yes, this hurts me a bit too. If you have a clean solution so I can convert this to a table-driven test, let me know.

Also, for code coverage, it only really measures the coverage from tests in the same package.

I'll think of a way to do code reuse. Let me know if it is too ugly.